src/functionalTest/resources/scenarios/RIA-412-RIA-1571-list-case-confirmation.json

```diff
@@ -7,15 +7,16 @@
       "eventId": "listCase",
       "state": "prepareForHearing",
       "caseData": {
-        "template": "minimal-appeal-submitted.json"
+        "template": "minimal-appeal-submitted.json",
+        "isAcceleratedDetainedAppeal": "No"
```

Comment:
Can you add another json (copy of this) where isAcceleratedDetainedAppeal is "Yes"? and then return the correct expectation body

Reply:
There is already one for ADA: minimal-ada-appeal-submitted.

Reply:
Also looking at it, this test is not ada specific. So this should stay the same with isAcceleratedDetainedAppeal = No to the template for default normal flow.
but we should have another test for ADA flow.